plugins/netlify/personal_access_token.go

```diff
@@ -0,0 +1,87 @@
+package netlify
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func PersonalAccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.PersonalAccessToken,
+		DocsURL:       sdk.URL("https://docs.netlify.com/cli/get-started/#authentication"),
+		ManagementURL: sdk.URL("https://app.netlify.com/user/applications#personal-access-tokens"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "Token used to authenticate to Netlify.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 43,
+					Prefix: "tGtp-",
+					Charset: schema.Charset{
+						Uppercase: true,
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer: importer.TryAll(
+			importer.TryEnvVarPair(defaultEnvVarMapping),
+			TryNetlifyConfigFile(),
+		)}
+}
+
+var defaultEnvVarMapping = map[string]sdk.FieldName {
+"NETLIFY_AUTH_TOKEN": fieldname.Token,
+}
+
+func TryNetlifyConfigFile() sdk.Importer {
+	return importer.TryFile("~/Library/Preferences/netlify/config.json", func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
```

Comment:
The location varies by the operating system:

https://docs.netlify.com/cli/get-started/#config-json-location

Could we update the importer to support OS-specific locations? `importer.MacOnly` and `importer.LinuxOnly` can help, there are some references of them in ngrok shell plugin:

https://github.com/1Password/shell-plugins/blob/8df60af1e69e89b17d11bac213375430b624f70d/plugins/ngrok/credentials.go#L51-L56